leetcode/993 Cousins in Binary Tree.cpp

```diff
@@ -23,18 +23,19 @@ struct TreeNode {
 class Solution {
 private:
     // <TreeNode*, parent-value>>
-    queue<tuple<TreeNode*, int>*> qLevels;
-    vector<tuple<TreeNode*, int>*> vCandidates;
+    queue<tuple<TreeNode*, int>*> qLevels; // you can omit having tuple if check childs instead of checking current node.
+    // check if left != null and left.val == x and same for right
+    vector<tuple<TreeNode*, int>*> vCandidates; // vector for a 0, 1, 2 values? better use array
```

Comment:
with vector it is more comfortable to check whether ther items are present in a vector. it is easier to check for size rather than try to access all elements and check them.